src/foam/nanos/crunch/ServerCrunchService.java

```diff
@@ -325,14 +326,15 @@ public UserCapabilityJunction updateJunction(
       ucj.setStatus(status);
     }
 
+    AuthService auth = (AuthService) x.get("auth");
     if (
-      subject.getRealUser().isAdmin()
+      auth.check(x, "warn.ucj.update")
```

Comment:
```suggestion
      auth.check(x, "usercapabilityjunction.warn.update")
```
like to keep the model first and thinking about other permissions like model.ro.property